cmd/tracee/cmd/root.go

```diff
@@ -212,6 +212,16 @@ func initCmd() error {
 		return errfmt.WrapError(err)
 	}
 
+	rootCmd.Flags().Int(
+		"pipeline-channel-size",
+		10000,
+		"<size>\t\t\t\tSize, in event objects, of each pipeline stage's output channel",
+	)
+	err = viper.BindPFlag("blob-perf-buffer-size", rootCmd.Flags().Lookup("blob-perf-buffer-size"))
```

Comment:
`blob-perf-buffer-size`? shouldn't this be `pipeline-channel-size`?

Reply:
It should be.